### ISSUE TITLE
Spike: Test integration with Jira sandbox API

### ISSUE THREAD

**Issue by CharliePatterson** [digital-land/submit#281]
## Context:

- LPAs have to submit their dataset endpoints to the platform by sending an email which opens a ticket in Jira.
- We've built a form to enable the collection of the required information in a more structured format, to ensure that all the relevant information is collected.
- This form needs to be able to send the data to Jira and open a ticket. Leveraging the Jira API is the best way to get this directly into Jira in the right format.
- We need to get approval from the cyber security team for sign-off on this approach, but as a first port of call we're going to test out the integration with the Jira sandbox.

## Acceptance criteria:

- As a data provider, when I complete the endpoint submission form, the information I have entered into the form is sent to the relevant Jira project
- As a member of the data management team, when I open a ticket in Jira which has been created by the endpoint submission form, I see all the relevant information inputted in the correct format

## Findings: https://docs.google.com/document/d/1tECxTpy2twzE2ZvKOCtrry4iym4vtqX67xwugeaEsQI/edit?tab=t.0

**Comment by GeorgeGoodall** [comment]
This ticket is blocked by: https://github.com/orgs/digital-land/projects/7/views/9?filterQuery=jira&pane=issue&itemId=74765094

**Comment by CharliePatterson** [comment]
Assigned to @CharliePatterson to get the API keys for the sandbox.

**Comment by GeorgeGoodall-GovUk** [comment]
putting this on hold until I can get access to jira

**Comment by GeorgeGoodall-GovUk** [comment]
It appears making a POST request to the Jira api is redirecting to a GET request. 
I briefly spoke with Jira's support team and they confirmed this seemed to be the case
a ticket has been raised with Jira support to have their engineering team look into this further
Until then I will set this ticket a blocked again

[https://support.atlassian.com/requests/PCS-346906/](https://support.atlassian.com.mcas.ms/requests/PCS-346906/?McasTsid=10618&McasCtx=4)
